src/components/ListRepositories.tsx

```diff
@@ -8,27 +8,23 @@ export declare interface ListRepositoriesProps {
   limit: number;
   handleLoadingMore: () => void;
   fetchedData: DbRepo[];
+  loading: boolean;
 }
 
 const ListRepositories = ({
   activeLink,
   limit,
   handleLoadingMore,
   fetchedData,
+  loading,
```

Comment:
`fetchedData` has a `meta` counterpart, unless we are using error and loading indicators in there, this here is o complication 🍕